pkg/client/kubectl_client.go

```diff
@@ -448,3 +466,31 @@ func (k *KubectlResourceOperations) authReconcile(obj *unstructured.Unstructured
 	}
 	return strings.Join(out, ". "), nil
 }
+
+type replacedStdin struct {
+	orig *os.File
+}
+
+func pumpToStdin(data io.Reader) (*replacedStdin, error) {
```

Comment:
could you please add a few comments explaining how the "trick" works ? I'm not 100% following all the swaps between the various variables ...